qa/L0_trace/opentelemetry_unittest.py

```diff
@@ -144,7 +178,11 @@ def _check_events(self, span_name, events):
             self.assertFalse(all(entry in events for entry in request_events))
             self.assertFalse(all(entry in events for entry in compute_events))
 
-        elif span_name == self.simple_model_name:
+        elif span_name in [
+            self.simple_model_name,
+            self.ensemble_model_name,
+            self.bls_model_name,
+        ]:
```

Comment:
Maybe move these into another attribute so there's only 1 place to update if new models are added later:
```suggestion
        elif span_name in self.test_models:
```

```
self.test_models = [
            self.simple_model_name,
            self.ensemble_model_name,
            self.bls_model_name,
        ]